src/Sulu/Bundle/SnippetBundle/Document/SnippetDocument.php

```diff
@@ -205,6 +206,22 @@ public function setLocale($locale)
     /**
      * {@inheritdoc}
      */
+    public function getOriginalLocale()
+    {
+        return $this->originalLocale;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setOriginalLocale($originalLocale)
+    {
+        $this->originalLocale = $originalLocale;
+    }
```

Comment:
That's another implication of https://github.com/sulu/sulu-document-manager/pull/82/files#r63675487. The snippet (at least currently) always has only one language, so ther is no need to distinguish between original locale and locale.